rocky/tools/models.py

```diff
@@ -161,6 +161,7 @@ class STATUSES(models.TextChoices):
 
     user = models.ForeignKey("account.KATUser", on_delete=models.PROTECT, related_name="members")
     organization = models.ForeignKey(Organization, on_delete=models.CASCADE, related_name="members")
+    groups = models.ManyToManyField(Group, related_name="members", blank=True)
```

Comment:
I feel like it would be logical that this defaults to 'client', since that's the group with the least permissions, and I think groups should never be empty. But I'm curious what @dekkers, @underdarknl or @praseodym think of this.

Reply:
Why don't we remove `blank=True` and always require a group to be set? I don't think we'd need a default, we should be able to ensure a group is always set.

Reply:
Yes, that's maybe even better. That enforces the user to make a decision. I like that suggestion.

Reply:
Here is decided that we can have a none group member.